ci/azure.yml

```diff
@@ -58,8 +58,12 @@ jobs:
           TARGET: mips-unknown-linux-musl
         mips64-unknown-linux-gnuabi64:
           TARGET: mips64-unknown-linux-gnuabi64
+        #mips64-unknown-linux-musl:
+        #  TARGET: mips64-unknown-linux-musl
```

Comment:
These tests need to be re-enabled

Reply:
Aren't `rustc` support necessary before the tests can even pass, forming a bootstrap dependency cycle?

Reply:
Well, seems like `libc` changes should go after the `librustc_target` enablement. I'll ready the `rustc` bits in a few hours.

Reply:
> Aren't rustc support necessary before the tests can even pass, forming a bootstrap dependency cycle?

Yes. First add the target to librustc_target. Then add support here, and add the targets to ci/build.sh (there is a nightly_nocore list there). Then update libc upstream in rust-lang/rust, and then you can start working on liballoc and libstd support while using libc. 

At some point in between you might want to enable these, but these might require libstd support before they are enabled, you might need to tune which tests are run for this target.